docs/integrations/sources/stripe.md

```diff
@@ -1,2 +1,46 @@
 # Stripe
 
+## Overview
+
+The Stripe source supports Full Refresh syncs. That is, every time a sync is run, Airbyte will copy all rows in the tables and columns you set up for replication into the destination in a new table.
+
+This Stripe source is based on the [Singer Stripe Tap](https://github.com/singer-io/tap-stripe).
+
+### Output schema
```

Comment:
thinking out loud: we shouldn't do this right now, but it would be nice to have a graphical representation like what FiveTran presents: https://docs.google.com/presentation/d/1p6KlVIZ8hi2ixtRcHbmZ9iGwNMq2he3ixaHuPuV7Gbw/edit#slide=id.p

Reply:
Maybe instead we can say that nested structures tend to be flattened into their own tables with a column inserted to point to the nested structure to which they belong? Right now, one needs to read a lot of json schema to understand what's going on

Reply:
I don't think this needs very detailed documentation, especially since it will be evident after discovery in the UI.